client/src/pages/Home.js

```diff
@@ -39,22 +45,29 @@ export function Home() {
 	return (
 		<main role="main">
 			<div>
+				<WeekChanger
```

Comment:
I don't think pulling things out into a WeekChanger component is the right idea. It adds a lot of complication for minimal gain

Also, I  tried to clarify that the position of the weekly buttons should not be over where Colette put them, but at the top of the table, specifically so you wouldn't have to do this :/

Reply:
Yeah I checked buttons ticket after reading this, because I thouth base on the my ticket's picture you need buttons top of logo. I was wrong. I'll fix it